includes/Modules/Analytics_4/Report/Request.php

```diff
@@ -380,4 +388,123 @@ protected function parse_dimension_filter( $dimension_name, $dimension_value ) {
 		return $filter_expression;
 	}
 
+	/**
+	 * Parses metric filters and returns a filter expression that should be added to the report request.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Data_Request $data Data request object.
+	 * @return Google_Service_AnalyticsData_FilterExpression The filter expression to use with the report request.
+	 */
+	protected function parse_metric_filters( Data_Request $data ) {
```

Comment:
Nice work here adding the metric filter support!

Please can you add some test coverage for this? See [here](https://github.com/google/site-kit-wp/blob/6bcba219b5abe67e5c1ceaa0ce6f96175ea0f98e/tests/phpunit/integration/Modules/Analytics_4Test.php#L878-L882) and [here](https://github.com/google/site-kit-wp/blob/6bcba219b5abe67e5c1ceaa0ce6f96175ea0f98e/tests/phpunit/integration/Modules/Analytics_4Test.php#L1012-L1030) for the existing dimension filter testing as a reference.